### DISCUSSION TITLE
Sublattice KKS Phase Field Model Example : Unable to find CrFe_sigma_out_var_0001.csv

### DISCUSSION THREAD

**Discussion by ppandit95** [idaholab/moose#28768] [Q&A Getting Started]
### Check these boxes if you have followed the posting rules.

- [X] I have consulted the [Posting Guidelines](https://github.com/idaholab/moose/discussions/18270).
- [X] I have searched the [Discussions Forum](https://github.com/idaholab/moose/discussions) and [MOOSE Framework Troubleshooting](https://mooseframework.inl.gov/help/troubleshooting.html) and have not found what I was looking for
- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)

### Issue or question about MOOSE

Hi Developers

I was trying to run sublattice KKS model which requires  CrFe_sigma_out_var_0001.csv file to generate the function requiredin the simulation but it seems that file is missing from the source code so could anyone direct me to desired location to get the file so that I can further work on utilizing this model for multi phase scenario.

Many Thanks
Pushkar

**Comment by GiudGiud** [comment]
@dschwen seems you contributed this example here
https://github.com/idaholab/moose/blob/9fb4f87899b841a2a4a440d625f842b4809d7ed0/modules/phase_field/examples/slkks/CrFe.i#L36

can you please provide the missing csv file?
